devnet/README.md

```diff
@@ -23,6 +23,40 @@ docker-compose down
 
 > **_NOTE:_**  The devnet is not stateful, so subsequent runs will start from the genesis block.
 
+## Running Cross Client Devnet
+
+A cross-client devnet is network of gossamer node(s) and substrate node(s).
+
+Steps to run a two node cross client devnet
+
+- Generate the `genesis-spec.json` file by running:
+
+```
+./polkadot build-spec --disable-default-bootnode --dev > genesis-spec.json
```

Comment:
maybe mention where/how to install polkadot or just link as a requirement for this.